src/odemis/gui/model/__init__.py

```diff
@@ -1084,7 +1083,7 @@ def __init__(self, main):
         if model.MD_POS_ACTIVE_RANGE not in md:
             raise KeyError("Stage has no MD_POS_ACTIVE_RANGE metadata.")
         carrier_range = md[model.MD_POS_ACTIVE_RANGE]
-        tlx, tly = float(carrier_range["x"][0]), float(carrier_range["y"][0])  # top left carrier position in m
+        minx, miny = float(carrier_range["x"][0]), float(carrier_range["y"][0])  # top left carrier position in m
```

Comment:
Why did you choose to not use an underscore between min_y? (and min_x?)

Reply:
We use "minx" and "miny" at other places in odemis in the same context, so I wanted to be consistent. My preference would have been for "xmin" and "ymin", but it doesn't really change much.